source/documentation/using_the_api/api_reference.md

```diff
@@ -272,7 +272,7 @@ Authorization: YOUR-API-KEY-HERE
 ```http
 HTTP/1.1 200
 Content-Type: application/json
-Link: <?start=101&limit=100>; rel="next"
+Link: <?start=1&limit=10>; rel="next"
```

Comment:
The response for `start=1&limit=10` is `Link: <?start=11&limit=10>; rel="next"`.

You can check the actual values for a response with curl with the flag `-I` (capital i):

```
curl -I "https://country.register.gov.uk/entries?start=1&limit=10"
```